extras/awslambda_image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM public.ecr.aws/lambda/python:3.7
+
+RUN curl -1sLf \
+    'https://repositories.timber.io/public/vector/cfg/setup/bash.rpm.sh' \
+    | bash
+
+RUN yum install -y vector wget
+
+RUN pip install awslambdaric metaflow awscli
```

Comment:
Do we need to install `awslambdaric` if the lambda python image is used? Curious if there is a way to somehow not require `runtime interface client` in the image - that way any arbitrary image can be used for both AWS Batch & AWS Lambda : say the entry point installs `awslambdaric` first and then invokes `python -m awslambdaric`?

Reply:
You're right, we probably don't have to install it if we use the Lambda python image.

As for using an arbitrary image and installing `awslambdaric` on the fly, it [seems like](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html#images-create-2) `awslambdaric` is not a pure Python package, so you'd have to have the gcc toolchain there as well. That seems like it could become pretty fragile. (I also wonder if AWS would kill the container prematurely if the runtime takes too long to initialize).

Reply:
Gotcha, I see, so the scenario you have in mind are users using Batch today and switching to Lambda?

One additional quirk to note here is that unlike e.g. Batch, your image for Lambda has to be a ECR image. Even [public ECR repo](https://docs.aws.amazon.com/AmazonECR/latest/public/public-repositories.html) doesn't cut it. 

So it seems like it would only benefit users that satisfy _all_ of the below conditions:
1) Use batch for these tasks today and want to move to `@awslambda`
2) ..and use custom images 
3) ..and store those images in ECR
4) ..and those images happen to have gcc toolchain to build awslambdaric

I'm not sure it is worth it, given that on-the-fly installation would be a bit hacky, and it would have to finish under undocumented deadline of what seems to be ~10s, ( [see this thread](https://stackoverflow.com/questions/55412624/does-aws-lambda-charge-for-the-time-spent-initializing-code/55426800) ) or possibly trigger more weird behaviors

Reply:
Discussed offline, we can't install awslambdaric on the fly as it has to be the entrypoint of the image - closing